python/pyspark/ml/torch/distributor.py

```diff
@@ -263,6 +265,10 @@ class TorchDistributor(Distributor):
 
     .. versionadded:: 3.4.0
 
+    .. versionchanged:: 3.5.0
+        Supports Spark Connect. Note local model with GPU is not supported yet, will be fixed
```

Comment:
nit: Note that...

Also, "local model" or "local mode"?

Reply:
good catch, will fix.